sklearn/cluster/hierarchical.py

```diff
@@ -902,9 +919,17 @@ class FeatureAgglomeration(AgglomerativeClustering, AgglomerationTransform):
     n_leaves_ : int
         Number of leaves in the hierarchical tree.
 
+    n_connected_components_ : int
+        The estimated number of connected components in the graph.
+
     n_components_ : int
```

Comment:
I'm okay for attributes to just be removed from documentation when deprecated

Reply:
Just FYI I actually told @scouvreur to add it since it's in the dev guide

Reply:
What would you prefer I do ?